DigitalLearningSolutions.Web/Controllers/NotificationPreferencesController.cs

```diff
@@ -65,10 +65,26 @@ private IEnumerable<NotificationPreference> GetNotificationPreferencesForUser(st
 
         [HttpPost]
         [Route("/NotificationPreferences/Edit/{userType}")]
-        public IActionResult SaveNotificationPreferences(IEnumerable<int> notifications)
+        public IActionResult SaveNotificationPreferences(string userType, IEnumerable<int> notifications)

```

Comment:
Could we name this "notificationIds" to keep it consistent with the naming below? :)